examples/GPU/example_learn_samples.py

```diff
@@ -54,7 +54,7 @@ def __init__(self, inital_trajectory):
             data=torch.Tensor(inital_trajectory),
             requires_grad=True,
         )
-        self.operator = get_operator("gpunufft", wrt_data=True, wrt_traj=True)(
```

Comment:
why do you prefer cufinufft ?

Reply:
The idea is to run cufinufft with density compensation, basically increase the coverage